AUTHORS.md

```diff
@@ -98,6 +99,12 @@ possible:
  * yilinwei
  * Zach Abbott
 
+Cats has been heavily inspired by [scalaz](https://github.com/scalaz/scalaz),
+and some Cats code is only a slightly modified version of code originating in
+scalaz. Therefore, we'd also like to credit and thank all of the
```

Comment:
This is nitpicky, but maybe capitalize "scalaz" throughout for consistency with "Cats" and other project names here, and with the Scalaz README?